scripts/demo-fruits.sh

```diff
@@ -10,51 +10,18 @@ source ${SCRIPTS_DIR}/play-demo.sh
 ####################################
 VM_IP=${VM_IP:=127.0.0.1}
 NAMESPACE=app
-REGISTRY=localhost:5000
-IMAGE_VERSION=latest
+IMAGE=quay.io/halkyonio/atomic-fruits:latest
 INGRESS_HOST=atomic-fruits.${VM_IP}.nip.io
-QUARKUS_APP_PATH="$HOME/atomic-fruits-service"
-GITHUB_REPO=https://github.com/aureamunoz/atomic-fruits-service
-
-# Database credential
-DB_NAMESPACE=db
-USERNAME=healthy
-PASSWORD=healthy
-TYPE=postgresql
 
 # Parameters to play the demo
 TYPE_SPEED=${TYPE_SPEED:=200}
 NO_WAIT=true
 
-if [[ ! -d "${QUARKUS_APP_PATH}" ]]; then
-  git clone ${GITHUB_REPO} ${QUARKUS_APP_PATH}
-fi
-
-if [[ "$1" == "clean" ]]; then
-  pe "helm uninstall postgresql -n ${DB_NAMESPACE} || true"
```

Comment:
Can we keep the option `clean` to uninstall the atomic-fruits and/or db hlm charts ?

Reply:
Readded.